storage-proofs/core/src/lib.rs

```diff
@@ -37,19 +38,3 @@ pub(crate) const TEST_SEED: [u8; 16] = [
 pub const MAX_LEGACY_POREP_REGISTERED_PROOF_ID: u64 = 4;
 
 pub type PoRepID = [u8; 32];
-pub fn is_legacy_porep_id(porep_id: PoRepID) -> bool {
```

Comment:
we should keep this around and validate that we have the correct porep id for now

Reply:
I can add this back, but I don't think asserting this around too often is the best idea, as the behavior is fully decoupled from the porep_id at the moment, and that would explicitly tie it back to it.

Reply:
I know, but we should have this check for these two API versions at the high level at least, maybe in filecoin-proofs, just to make sure we maintain the same invariants that we have right now.